index.html

```diff
@@ -56,7 +71,7 @@ <h3 class="connect_text">LET'S CONNECT</h3>
         </section>
         <img src="./assets/Ellipse_11.png" class="left_ellipse" alt="ellipse">
         <img src="./assets/Ellipse_11.png" class="bottom_ellipse" alt="ellipse">
-        <img src="./assets/Rectangle_6.png" class="left_rectangle" alt="rectangle">
+        <img src="./assets/Ellipse_3.png" class="right_ellipse" alt="rectangle">
```

Comment:
- [ ] [OPTIONAL] Please avoid adding background images with the `img` tag. Rather add them in your `CSS` file using the `background-image` property